sirepo/template/opal.py

```diff
@@ -116,6 +116,22 @@ def is_ignore_field(self, field):
         return field == "name" or field == self.IS_DISABLED_FIELD
 
 
+class _OpalLogParser(template_common.LogParser):
+    def _parse_log(self, file_path):
+        res = ""
+        visited = set()
+        with pkio.open_text(file_path) as f:
+            for line in f:
+                if re.search(r"^Error.*?>\s*[\w\"]", line):
```

Comment:
I think the part above should be in the base class - line parsers should just be given a line of input, and return any errors parsed out of it. The superclass should collect the errors and de-dup them.